huaweicloud/services/evs/resource_huaweicloud_evs_volume.go

```diff
@@ -307,6 +322,36 @@ func resourceEvsVolumeCreate(ctx context.Context, d *schema.ResourceData, meta i
 	return resourceEvsVolumeRead(ctx, d, meta)
 }
 
+func waitEvsJobSuccess(ctx context.Context, client *golangsdk.ServiceClient, jobId string,
+	timeout time.Duration) error {
+	stateConf := &resource.StateChangeConf{
+		Pending:      []string{"RUNNING"},
+		Target:       []string{"SUCCESS"},
+		Refresh:      evsJobRefreshFunc(client, jobId),
+		Timeout:      timeout,
+		Delay:        5 * time.Second,
+		PollInterval: 10 * time.Second,
+	}
+	_, err := stateConf.WaitForStateContext(ctx)
+	return err
+}
+
+func evsJobRefreshFunc(c *golangsdk.ServiceClient, jobId string) resource.StateRefreshFunc {
+	return func() (interface{}, string, error) {
+		resp, err := evsjobs.GetJobDetails(c, jobId).ExtractJob()
+		if err != nil {
+			// there has no special code here
+			return resp, "", err
```

Comment:
```suggestion
			return resp, "ERROR", err
```
the empty string will cost the customer confuse,: `expect 'SUCCESS', but ''`

Reply:
Done